support-models/src/main/scala/com/gu/support/catalog/Product.scala

```diff
@@ -67,76 +69,137 @@ case object Contribution extends Product {
 }
 
 case object Paper extends Product {
-  private lazy val ukOnly = List(CountryGroup.UK)
+
+  private def collection(productRatePlanId: ProductRatePlanId, productOptions: ProductOptions, description: String): ProductRatePlan[Paper.type] =
+    ProductRatePlan(productRatePlanId, Monthly, Collection, productOptions, description, List(CountryGroup.UK))
+
+  private def homeDelivery(productRatePlanId: ProductRatePlanId, productOptions: ProductOptions, description: String): ProductRatePlan[Paper.type] =
+    ProductRatePlan(productRatePlanId, Monthly, HomeDelivery, productOptions, description, List(CountryGroup.UK))
+
+  private lazy val useDigitalVoucher = false
```

Comment:
This flag determines whether we use the new digital voucher product or the old paper one.